src/main/java/com/patchworkmc/patch/LevelGeneratorTypeTransformer.java

```diff
@@ -27,15 +27,6 @@ public MethodVisitor visitMethod(int access, String name, String descriptor, Str
 			super(Opcodes.ASM7, methodVisitor);
 		}
 
-		@Override
-		public void visitTypeInsn(int opcode, String type) {
```

Comment:
Why is this removed? As far as I can tell, it's needed for level generator types to properly work.